__tests__/api.test.js

```diff
@@ -123,3 +126,50 @@ describe('GET /api/articles', () => {
     })
 }); 
 
+describe('GET /api/articles/:article_id/comments', () => {
+    test('responds with an array of comments for a given article ID with the correct properties, with the most recent comment first', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+        const { comments } = response.body;
+        expect(comments).toHaveLength(11)
+
+        comments.forEach((comment) => {
+            console.log(comment)
+        expect(typeof comment.comment_id).toBe("number")
+        expect(typeof comment.votes).toBe('number')
+        expect(typeof comment.created_at).toBe('string')
+        expect(typeof comment.author).toBe('string')
+        expect(typeof comment.body).toBe('string') 
+        expect(typeof comment.article_id).toBe('number')
```

Comment:
We can be a lot more specific here - since we only want to get comments from one particular article, we can check whether the id matches in the comments